modules/ROOT/pages/edifact-send-settings.adoc

```diff
@@ -0,0 +1,170 @@
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Default
+|Authorization qualifier (ISA01)
+|
+|None
+|Security qualifier (ISA03)
+|
+|None
+|===
+
+
+[[ta1-ack-settings]]
+== TA1 Acknowledgment Settings
+
+Use the following settings to configure whether a technical acknowledgment is required, and the receive endpoint for the acknowledgment:
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Default
+
+|Expect TA1 for each transmission sent to <partner>
+a|If selected, sets the value in ISA14 to 1 on the generated EDIFACT payload, indicating that a Technical acknowledgment (TA1) is required.
+|Disabled
+|Endpoint
+a|When the option to expect a TA1 is selected, click *Select* to specify the *Receive from <partner>* endpoint. The acknowledgment will be received as an inbound transmission.
+|None
+|===
+
+[[global-control-numbers]]
+== Global Control Numbers
+
+Partner Manager applies a global control number at the partner level on the EDIFACT interchanges and functional groups generated from outbound message flows. These control numbers are sequenced across all message types (including functional acknowledgments) that are sent to a trading partner.
+
+The control number starts with a default value of `1`, and is sequentially increased as your organization begins sending outbound transmissions.
+
+If you need to set the control number sequence value to a different number to maintain sequencing and avoid duplication:
+
+. Click *Change* and enter the desired control number.
+. Click *Save*.
+
+[NOTE]
+It can take up to five minutes to apply the change at runtime.
+
+[[character-set]]
+== Character Set and Separators
+
+These settings enable you to set the character encoding and delimiters to use on the generated EDIFACT payload:
+
+[%header%autowidth.spread]
+|===
+|Setting |Description | Default Value
+
+|Character set
+a|Characters allowed in string data.
+
+Possible values include:
+
+* *Basic*
+* *Extended*
+* *Unrestricted* | `Extended`
+
+|Character encoding
+a|Character encoding for messages.
+
+Possible values include:
+
+* *PLATFORM*
+* *ASCII*
+* *ISO8859_1*
+* *IBM1047*
+* *UTF8* | `UTF8`
+
+|Segment terminator
+a|Segment terminator character.
+| `~`
+
+|Data element separator
+a|Data element separator character.
+| `*`
+
+|Component element separator (ISA16)
+a|Component element separator character.
+| `>`
+
+|Line ending between segments
+a|Line ending to add between segments. This enables you to add line endings between segments to improve the readability of the output message text.
+
+Possible values include:
+
+* *NONE* +
+EDIFACT payload generated will be a single line, with segments separated by the segment terminator character
+* *CR* +
+Carriage Return
+* *CRLF* +
+Carriage Return & Line Feed
+* *LF* +
+Line Feed
+|`CRLF`
+|===
+
+== See Also
+
+* xref:create-outbound-message-flow.adoc[Create and Configure an Outbound Message Flow]
```

Comment:
@dhirsch1 - this topic needs to be replaced with content from https://docs.google.com/document/d/16J6CubEMk1mAnr5JWOtRFMjIshCA6xnQthYg3Je4yMA/edit#